src/Nethermind/Nethermind.EthStats/EthStatsPlugin.cs

```diff
@@ -1,118 +1,27 @@
 // SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
-using System;
 using System.Threading.Tasks;
+using Autofac;
+using Autofac.Core;
 using Nethermind.Api;
 using Nethermind.Api.Extensions;
-using Nethermind.Core;
-using Nethermind.Core.Crypto;
-using Nethermind.EthStats.Clients;
-using Nethermind.EthStats.Configs;
-using Nethermind.EthStats.Integrations;
-using Nethermind.EthStats.Senders;
-using Nethermind.Network.Config;
-using Nethermind.Network.P2P;
+using Nethermind.Api.Steps;
 
 namespace Nethermind.EthStats;
 
 public class EthStatsPlugin(IEthStatsConfig ethStatsConfig) : INethermindPlugin
 {
-    private IEthStatsClient _ethStatsClient = null!;
-    private IEthStatsIntegration _ethStatsIntegration = null!;
-    private INethermindApi _api = null!;
-    private Logging.ILogger _logger;
-
-    private readonly bool _isOn = ethStatsConfig.Enabled;
-
-    public bool Enabled => _isOn;
-
+    public bool Enabled => ethStatsConfig.Enabled;
     public string Name => "EthStats";
     public string Description => "Ethereum Statistics";
     public string Author => "Nethermind";
+    public ValueTask DisposeAsync() => ValueTask.CompletedTask;
+    public Task Init(INethermindApi nethermindApi) => Task.CompletedTask;
```

Comment:
Maybe those should be default implementations in the interface?

Reply:
Removed init. Dispose don't have default though.